discovery-provider/scripts/circle-ci.sh

```diff
@@ -5,6 +5,8 @@ function main {
   set -e
   export audius_ipfs_port=5001
   export audius_redis_url=redis://localhost:6379/0
+  export audius_delegate_owner_wallet=0x1D9c77BcfBfa66D37390BF2335f0140979a6122B
+  export audius_delegate_private_key=0x3873ed01bfb13621f9301487cc61326580614a5b99f3c33cf39c6f9da3a19cad
```

Comment:
even here...how does this work exactly?

Reply:
i think the wallet/priv key pair work even if i restart the local system?